test/test_requestor.py

```diff
@@ -99,14 +90,3 @@ def test_connection_error(self):
       connection.experiments(1).fetch()
     e = e.value
     assert str(e) == 'ConnectionException: fake connection exception'
-
-  def test_unicode_json(self):
```

Comment:
chose to remove this because we wouldn't ever want to return a `bytes`-like object for these exceptions; and `unicode` is same as `str` so we no longer need to distinguish in Python3